src/mlx90632_extended_meas.c

```diff
@@ -0,0 +1,366 @@
+/**
+ * @file mlx90632.c
```

Comment:
you can omit filename for doxygen nowdays and let it figure out itself. Would be nice copy/paste here if I would do that in past.

Reply:
```suggestion
 * @file
```